.circleci/config.yml

```diff
@@ -4,9 +4,18 @@
 version: 2
 jobs:
   verify:
-    docker:
-      - image: circleci/golang:1.13.1
+    machine:
+      image: circleci/classic:latest
     steps:
+      - run:
+          name: Install Go
+          command: |
+            sudo rm -rf /usr/local/go
+            curl -sSL "https://dl.google.com/go/go1.13.1.linux-amd64.tar.gz" -o /tmp/go.tar.gz
+            echo "94f874037b82ea5353f4061e543681a0e79657f787437974214629af8407d124  /tmp/go.tar.gz" | sha256sum -c
+            sudo tar -xzf /tmp/go.tar.gz -C /usr/local/
+            echo "export PATH=$PATH:/usr/local/go/bin" >> $BASH_ENV
+            echo 'export GOPATH=$HOME/go' >> $BASH_ENV
```

Comment:
Not against this change, but why was it needed?

Reply:
`make verify` now needs access to Docker (so it can build `protoc` and the Go plugin in a Docker container, and then execute that container).

I tried to do the remote docker context mount thing that CircleCI can do, but I also need to mount a local directory (with the source code in it). My only way to do this would be to copy the code to this remote docker context, and then copy it back after.

So after all of that it's easier to just use a machine instead of a container so I can use Docker directly.